product-histories.md

```diff
@@ -16,32 +18,40 @@ Fast forward to 2014. We'd been feeling pulled in too many directions for too lo
 
 When 37signals became Basecamp in 2014, we decided to spin off or sell our other (very successful!) applications. One of those was Highrise, our customer relationship management (CRM) tool. Highrise became a subsidiary of Basecamp run by its own board, executive, and staff. In 2018, we took back control of Highrise, disbanded the separate company, and turned off new signups. Highrise is now, as Basecamp Classic, Basecamp 2, Backpack, and Campfire, part of our until-the-end-of-the-internet legacy. Maintained for existing customers, but no longer developed for new features or new customers.
 
-## Campfire
-
-In 2006, we built Campfire to help businesses communicate better. It’s a simple, real-time web-based group chat tool that lets people set up password-protected chat rooms super quickly. At the very beginning it didn’t integrate with Basecamp, but when it did it took off. Signups for Campfire were closed in February 2017.
-
-## Know Your Company
+## Ta-da List
 
-In 2013, Basecamp created a software tool that allows managers to ask questions to gain insight into what their employees think about work, the culture, the direction of the company, what people are working on, or even something personal like their favorite recent movie. We created KYC because when the Basecamp team grew to about 30 people, Jason realized he didn't know the company or its people as well as he should have. So, KYC was created initially as an internal tool that eventually became a product that we sold to customers. In 2014, we [sold Know Your Company](https://knowyourcompany.com/story) to Claire Lew, and she's been running it ever since!
+Ten years ago, to-do list apps were few and far between. So, we created Ta-Da List as a stand alone free subscription app at tadalist.com in January 2005. Now that people have to-do lists on their phones, desktops, laptops, tablets, and refrigerator doors, we recognized that Ta-Da List was a little outdated. We shuttered tadalist.com in 2014, but existing customers were able to retain their accounts.
```

Comment:
Think we should be clear that for us "shuttered" always simply means "closed for new customers". Sounds like it stopped working.